example/axisnow/decryptAndEmbed.ts

```diff
@@ -1,17 +1,35 @@
-import Decryptor from '@nypl-simplified-packages/axisnow-access-control-web';
-
 type DecryptChapter = (resourceUrl: string) => Promise<string>;
 
 type DecryptorParams = {
   isbn: string;
   book_vault_uuid: string;
 };
 
+type IDecryptorInstance = {
```

Comment:
I still don't understand why this goes under /example/ rather than in /src/  Wouldn't all Decryptors, be they AxisNow or not, share the same interface?